Sources/Main.cpp

```diff
@@ -36,6 +37,7 @@ TX584Form *X584Form;
 __fastcall TX584Form::TX584Form(TComponent* Owner)
     : TForm(Owner), CPU(16), OpFilter(0), ResFilter(-1), ResButton(NULL), SelCount(0), ClipboardSize(0)
 {
+	ClipboardFormat = RegisterClipboardFormatA("X584Clipboard");
```

Comment:
Let's use the Unicode version `RegisterClipboardFormatW` to be consistent with other WinAPI functions.